models/gyroscope/arbitrum/gyroscope_arbitrum_schema.yml

```diff
@@ -0,0 +1,94 @@
+version: 2
+
+models:
+
+  - name: gyroscope_arbitrum_trades
+
+    meta:
+
+      blockchain: arbitrum
+      project: gyroscope
+      contributors: fmarrr
+
+    config:
+
+      tags: ["arbitrum", "gyroscope", "trades"]
+
+    description: trades on gyroscope E-CLP pools on arbitrum
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - token_bought_amount
+            - token_sold_amount
```

Comment:
these two shouldn't be here
please be sure columns in this list match the unique keys in the model exactly

Reply:
this will apply to all blockchain versions of the spell. i'll only comment here, but go through each and finalize a universal unique key set and then apply to all schema files